src/private/php/Assigner.php

```diff
@@ -162,5 +162,34 @@ public static function canUseAssignerSgArray(array $serverGroups): bool {
 
         return false;
     }
+    
+    public static function getCooldownPeriod(): int {
+        return Config::get("assigner_cooldown_seconds");
+    }
+
+    public static function isOnCooldown(): array {
+        $cooldownPeriod = self::getCooldownPeriod();
+        $cldbid = Auth::getCldbid();
+        $cacheKey = "assigner_last_usage_" . $cldbid;
+        $lastUsageTime = CacheManager::i()->get($cacheKey) ?? 0;
```

Comment:
CacheManager is used for caching parts of the TeamSpeak server data. Please create a new cache instance for use with assigner only, similarly to Auth/login codes: https://github.com/Wruczek/ts-website/blob/042c0bfec77e55535362550fcc38d7c919171556/src/private/php/Auth.php#L126-L145